### PR TITLE
Version 1.0 of Corda Docker image

### PR DESCRIPTION
**Pull request by przemolb**
It has been tested (I have built running docker image).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,27 +2,28 @@
 FROM phusion/baseimage:0.9.22
```
Comment (benabineri) [inline Dockerfile]: Should we update this? (I used debian:9 in my docker-compose example.)

File: README.md [hunk 2]
```diff
@@ -37,4 +37,4 @@ With docker environment you can not only control Corda node set up but also pass
 
 For example following incantation create container with JMX enabled, assuming that you called Docker image _corda:m11_.Please note that JMX over RMI was disable in version M12, so following command wont work.
```
Comment (benabineri) [inline README.md]: This should be updated or removed.

File: corda-1.0.0.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+export JAVA_CAPSULE="${JAVA_CAPSULE-''}"
+
+cd /opt/corda
+
+cat > node.conf << EOF
+basedir : "/opt/corda"
+p2pAddress : "$CORDA_HOST:$CORDA_PORT_P2P"
+rpcAddress : "$CORDA_HOST:$CORDA_PORT_RPC"
+h2port : 11000
+myLegalName : "CN=$CORDA_LEGAL_NAME,O=$CORDA_ORG,OU=$CORDA_ORG_UNIT,L=$CORDA_CITY,C=$CORDA_COUNTRY, E=$CORDA_EMAIL"
```
Comment (benabineri) [inline corda-1.0.0.sh]: This isn't a valid legal name in V1.

File: corda-1.0.0.sh [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+keyStorePassword : "cordacadevpass"
+trustStorePassword : "trustpass"
+extraAdvertisedServiceIds: [ "" ]
+useHTTPS : false
+devMode : true
+rpcUsers=[
+    {
+        user=corda
+        password=not_blockchain
```
Comment (benabineri) [inline corda-1.0.0.sh]: Use a different password please - RGB doesn't like this one. (It reminds people of the "Corda isn't blockchain" discussion.)

File: Dockerfile (hunk 1 above)
Comment (przemolb) [reply Dockerfile]: From https://github.com/phusion/baseimage-docker 
This image is prepared for startups, hobbyist, single developers interesting in running it on their own laptops etc and personally I quite like it.
In a close future I can prepare more Dockerfiles for different distros - from my experience each company have different policy and if company X (finance ?) prefers RH they will rather refuse other distribution.

File: corda-1.0.0.sh (hunk 3 above)
Comment (przemolb) [reply corda-1.0.0.sh]: Valid point - will have a look.

File: corda-1.0.0.sh (hunk 4 above)
Comment (przemolb) [reply corda-1.0.0.sh]: Will change this.

**Comment by przemolb** [comment]
Have just pushed new stuff.

File: Dockerfile (hunk 1 above)
Comment (benabineri) [reply Dockerfile]: 0.9.22 is the latest

**Pull request merged**